microsoft-edge/devtools-guide-chromium/customize/theme.md

```diff
@@ -0,0 +1,78 @@
+---
+description: How to apply different color themes to Microsoft Edge DevTools.
+title: Apply color themes to Microsoft Edge DevTools
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 07/19/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+# Apply color themes to Microsoft Edge DevTools  
+
+Various color themes can be applied to the DevTools. Themes affect the color of panes, buttons, and code syntax highlighting. Starting with [Microsoft Edge 93][WhatsNew93], DevTools offers several new themes from [Visual Studio Code][VSCode] such as Monokai and Solarized Dark.  
+
+:::image type="complex" source="./media/all-devtools-themes.png" alt-text="All DevTools color themes" lightbox="./media/all-devtools-themes.png":::
+   All of the available DevTools color themes 
+:::image-end:::  
+
+> [!NOTE]
+> Prior to [Microsoft Edge 93][WhatsNew93], DevTools only had a light and dark theme.  
+
+## How to change themes  
+
+By default, the DevTools theme is set to **System preferred**. This setting selects the Light+ or Dark+ theme for DevTools, depending on your operating system's default color theme. However, you can change this default to one of several other themes:  
+
+Light themes:  
+- Light+ (Default)  
+- Chromium Light  
+- Quiet Light  
+- Solarized Light  
+
+Dark themes:  
+- Dark+ (Default)  
+- Abyss  
+- Chromium Dark  
+- Kimbie Dark  
+- Monokai  
+- Monokai Dimmed  
+- Red  
+- Solarized Dark  
+- Tomorrow Night Blue  
+
+Theme options appear in DevTools **Settings**.
+
+1.  Choose **Settings** (cog icon) from the toolbar.  You may also open **Settings** by selecting `F1`.  
+
+    :::image type="complex" source="./media/setting-button.png" alt-text="Open Settings" lightbox="./media/setting-button.png":::
+       Open **Settings**  
+    :::image-end:::  
+
+1.  On the **Preferences** pane,  under **Appearance**, choose a theme from the Theme dropdown.  
+    
+    :::image type="complex" source="./media/customize-theme-setting.png" alt-text="Preferences" lightbox="./media/customize-theme-setting.png":::
+       Selecting a theme in **Preferences**  
+    :::image-end:::  
+
+## Changing theme from the command menu  
```

Comment:
It seems odd that "Changing theme from the command menu" has a header but the content on changing theme from the Settings menu does not. Should we either remove this heading or make parallel ###-subheadings (e.g., "Changing theme from Settings" and "Changing theme from the command menu")?

Reply:
Ah, good catch. It does indeed feel weird. I'm leaning toward adding another sub-heading for changing theme from the settings, since there aren't many headings in this page yet.